sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -1134,6 +1138,14 @@ export class MessageSession extends LinkEntity {
       const nextRenewalTimeout = calculateRenewAfterDuration(this.sessionLockedUntilUtc!);
       this._sessionLockRenewalTimer = setTimeout(async () => {
         try {
+          let receiverName;
+          if (this._receiver) {
+            receiverName = this._receiver.name;
```

Comment:
The right place to set the `receiverName` would be right after the receiver is created. Otherwise, this property will have no value if user has disabled auto lock renewal. So, can we move this to inside `init()`?

Reply:
Looks like the local variable name was confusing.
The receiverName was being set in create, look at near line # 1213 in same file.
But moved it to within init() as recommended.